.github/pull_request_template.md

```diff
@@ -1,48 +1,59 @@
 <!--
 ### How to use
-Leave the headings unless they don't apply to your PR, replace commented out text (surrounded with <!–– and ––>) with text describing your PR.
-NOTE: Please grant permission for repository maintainers to edit your PR.
-It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them.
+Leave the headings unless they don't apply to your PR, and replace commented out text (surrounded with <!–– and ––>) with text describing your PR.
+
+NOTE: Please grant permission for repository maintainers to edit your PR.  It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them.
 -->
 
 #### Summary
 <!--
-A one-line description of your change that will be extracted and added to the [project changelog](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt).
+This section should consist of exactly one line, formatted like this:
+
+`SUMMARY: [Category] "[Briefly describe the change]"`
+
+The backquotes `` are optional, and you should not enter the square brackets [].  Category must be one of these:
```

Comment:
Remove the `The backquotes `` are optional` section, it's needless here.

Reply:
Are they not optional?

Reply:
They are **purely** optional, so they don't worth mentioning at all.

Reply:
That simplifies things, thanks. I removed them from the latest iteration.